packages/core/strapi/lib/commands/actions/plugin/build-command/command.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const { getLocalScript } = require('../../../utils/helpers');
+
+/**
+ * `$ strapi plugin:build`
+ * @param {import('../../../../types/core/commands').AddCommandOptions} options
+ */
+module.exports = ({ command }) => {
+  command
+    .command('plugin:build')
+    .description('Bundle your strapi plugin for publishing.')
+    .option('-y, --yes', 'Skip all confirmation prompts', false)
```

Comment:
In DTS we use --force, --yes does make sense in this context, I just want to make sure we're clear on the difference in meanings so that we don't duplicate a term with identical usage.

Reply:
Ah thank you for highlighting, I think `force` is fine to use here and it avoids a split in terminology.